mleap-xgboost-spark/src/test/scala/ml/dmlc/xgboost4j/scala/spark/mleap/XGBoostRegressionModelParitySpec.scala

```diff
@@ -36,7 +36,7 @@ class XGBoostRegressionModelParitySpec extends FunSpec
   private val xgboostParams: Map[String, Any] = Map(
     "eta" -> 0.3,
     "max_depth" -> 2,
-    "objective" -> "reg:linear",
+    "objective" -> "reg:squarederror",
```

Comment:
just for my understanding, is this change required?

Reply:
yes, otherwise this test will fail.